src/main/scala/com/github/flakm/InMemoryLdapServer.scala

```diff
@@ -97,10 +98,24 @@ object InMemoryLdapServer {
     val ds: InMemoryDirectoryServer = new InMemoryDirectoryServer(
       settings
     )
+    object Converter {
+      import scala.jdk.CollectionConverters._
+      val allFilesFromConfiguration =
+        config.getStringList("inmemoryldap.files").asScala.toList
+    }
+    lazy val constructPathsFromFiles: List[String] => List[String] =
+      aListOfFileNames => {
+        aListOfFileNames
+          .map(
+            aSingleFileName => getClass().getResource(aSingleFileName).getPath
```

Comment:
`java.lang.Class#getResource` returns null if the reference with that name doesn't exist. Maybe we could throw a simple Exception in this case? It would probably be more handy to fail fast instead.

Reply:
I think that if user defines file that should be loaded to ldap during test and its missing in runtime there should be explicit exception thrown, don't you agree?

Reply:
oki doki! updated the proposal